test/modules/standard/Spawn/spawn-term.chpl

```diff
@@ -0,0 +1,9 @@
+use Spawn;
+use Time;
+
+var sub = spawn(["sleep", "10"]);
+sleep(1);
+sub.terminate();
+sub.poll();
+assert(sub.running == false);
+assert(sub.exit_status != 0);
```

Comment:
Same feedback as the other test - check for -SIGKILL and increase the sleep time.